compose/cli/main.py

```diff
@@ -141,12 +141,20 @@ def kill(self, project, options):
         Usage: kill [options] [SERVICE...]
 
         Options:
-            -s SIGNAL         SIGNAL to send to the container.
-                              Default signal is SIGKILL.
+            -s SIGNAL                  SIGNAL to send to the container.
+                                       Default signal is SIGKILL.
+            --max-workers NUM_WORKERS  Specify max number of workers to use to stop containers.
+                                       (default: 5)
+
+
         """
         signal = options.get('-s', 'SIGKILL')
+        max_workers = float(options.get('--max-workers') or DEFAULT_MAX_WORKERS)
```

Comment:
Yes, good spot, thanks. I'm re-working these right now anyway and moving max_workers out into an ENV variable.